torch/csrc/deploy/test_deploy.cpp

```diff
@@ -378,3 +378,30 @@ TEST(TorchpyTest, UsesDistributed) {
     I.self.attr("import_module")({"uses_distributed"});
   }
 }
+
+TEST(TorchpyTest, Autograd) {
```

Comment:
Was this test actually failing before your fix?
This would only fail if it runs on a GPU machine. Because ootherwise, the worker threads are never created.
Is that ok?

Reply:
re: the GPU thing, yeah it's fine for us, we only enable deploy tests on GPU builds currently anyway.

Reply:
ok!